### PR TITLE
LPS-183329 | Poshi Automation for Bug | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-161638 - Poshi Automation for Bug

Hi! :)
This pr is corresponding to this ticket: [LPS-175090](https://issues.liferay.com/browse/LPS-175090)  → [LPS-183329](https://issues.liferay.com/browse/LPS-183329)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 1]
```diff
@@ -1233,4 +1233,71 @@ definition {
...
+
+		task ("Then the copy of the page must come with the same permissions as the original") {
+			User.logoutPG();
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/${blankSiteName}/copy-content-page-name");
+
+			ContentPages.viewFragmentText(
+				fragmentName = "paragraph",
+				id = "element-text",
+				text = "Page Not Found");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1299]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 2]
```diff
@@ -1233,4 +1233,71 @@ definition {
...
+				key_roleTitle = "Guest",
+				locator1 = "Permissions#CONTENT_PERMISSIONS_VIEW_CHECKBOX");
+
+			Button.clickSave();
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Copy Content Page Name",
+				siteName = ${randomSiteName});
+
+			PageEditor.publish();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1288]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 3]
```diff
@@ -1233,4 +1233,71 @@ definition {
...
+
+			Button.clickAdd();
+
+			PermissionsInline.gotoPagePG(
+				pageName = "Copy Content Page Name",
+				site = ${randomSiteName});
+
+			AssertNotChecked.assertNotCheckedNotVisible(
+				key_roleTitle = "Guest",
+				locator1 = "Permissions#CONTENT_PERMISSIONS_VIEW_CHECKBOX");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1280]: Move to Then task block

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 4]
```diff
@@ -1233,4 +1233,62 @@ definition {
...
+
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONTENT_PERMISSIONS_VIEW_CHECKBOX",
+				roleTitle = "Guest");
+
+			SelectFrame.selectFrameTop();
+
+			Click(
+				key_modalTitle = "Permissions",
+				locator1 = "Button#CLOSE_MODAL");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1256]: IFrame.closeFrame

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 5]
```diff
@@ -1233,4 +1233,62 @@ definition {
 		}
 	}
 
+	@description = "This is a test for Bug LPS-175090. Permissions are copied when a Page is created as a copy of another Page"
+	@priority = 4
+	test ViewPermissionsPersistedInCopyPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1238]: Move to PagesAdmin.testcase

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdmin.testcase [hunk 6]
```diff
@@ -2205,4 +2205,65 @@ definition {
...
+				menuItem = "Permissions",
+				pageName = "Test Content Page Name");
+
+			IFrame.selectPermissionsFrame();
+
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONTENT_PERMISSIONS_VIEW_CHECKBOX",
+				roleTitle = "Guest");
+
+			SelectFrame.selectFrameTop();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdmin.testcase:2229]: No need

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/pull/12144
@rafaellajordao

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**